modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1595,6 +1595,30 @@ a|a double which is in the range `0.0` to `1.0`
 m|+++0.75+++
 |===
 
+
+[[config_server.cypher.parallel.worker_limit]]
+=== `server.cypher.parallel.worker_limit`
+
+label:enterprise-edition[Enterprise Edition]
+label:new[Introduced in 5.13]
+
+.server.cypher.parallel.worker_limit 
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a| Number of threads to allocate to Cypher worker threads for the parallel runtime.
+If set to a positive number, that number of workers will be started.
+If set to `0`, one worker will be started for every logical processor available to the Java Virtual Machine.
+
+If set to a negative number we will subtract the value from the number of logical processors available; for example, say Neo4j is running on a server with 16 available processors, using `server.cypher.parallel.worker_limit = -1` would then mean that 15 threads are available for the parallel runtime.
```

Comment:
```suggestion
If set to a negative number, that number will be subtracted from the total number of logical processors available on the server. For example, if the server has 16 available processors and you set `server.cypher.parallel.worker_limit` to `-1`, the parallel runtime will have 15 threads available.
```

Reply:
Thanks @renetapopova. I agree that it is slightly awkwardly phrased. However, I took it verbatim from the mono-repo, and changing it will mean there is a discrepancy between docs and the mono-repo. I propose that I change it in the mono-repo once 5.13 has been released and then update the docs accordingly (and that we therefore merge this as is). What do you think?

Reply:
Sure.

Reply:
I'll create a card so as not to forget and merge this one.